packages/sanity-web/src/components/SanityImageBase.tsx

```diff
@@ -0,0 +1,196 @@
+import type { ImageUrlBuilder, ImageCrop, ImageHotspot } from "sanity";
+
+import { getImageDimensions } from "@sanity/asset-utils";
+import imageUrlBuilder from "@sanity/image-url";
+import React from "react";
+
+export type SanityImageBaseProps = {
+  /** The aspect ratio of the image, in the format of `width/height`.
+   *
+   * @example
+   * ```
+   * <SanityImage data={image} aspectRatio="4/5" />
+   * ```
+   */
+  aspectRatio?: string;
+  config: {
+    dataset: string;
+    projectId: string;
+  };
+  className?: string;
+  data: {
+    _type: "image";
+    alt?: string;
+    asset?: {
+      _ref: string;
+      _type: "reference";
+    };
+    crop?: ImageCrop;
+    hotspot?: ImageHotspot;
+  } | null;
+} & React.ComponentPropsWithRef<"img">;
+
+const SanityImageBase = React.forwardRef<
```

Comment:
We should call it `SanityImage`.Then when you create a SanityImage component in a project you can import it with a SanityImageBase alias.

Reply:
Done